input/intro-notes/StructureDefinition-au-patient-intro.md

```diff
@@ -37,7 +37,7 @@
     <td>"document"</td>
     <td>SHALL use an appropriate value from <a href="ValueSet-rsg-source-document-type.html">AU Recorded Sex or Gender (RSG) Source Document Type</a> value set if any of the codes within the value set can apply to the concept being communicated, e.g. "passport".</td>
     <td>The name of the field within the source document where this information is recorded, e.g. "Sex".</td>
-    <td>SHALL use an appropriate value from <a href="ValueSet-au-jurisdiction-extended.html">Jurisdiction - AU Extended</a> value set if any of the codes within the value set can apply to the concept being communicated. If representing an Australian document use "AU" or the applicable state or territory code.</td>
+    <td>SHALL use an appropriate value from <a href="ValueSet-au-jurisdiction-extended.html">Jurisdiction ValueSet - AU Extended</a> if any of the codes within the value set can apply to the concept being communicated. If representing an Australian document use "AU" or the applicable state or territory code.</td>
```

Comment:
replace the word "value set" after the hyperlink closes. Yes it does your head in.

Reply:
I have re-added "value set" after the link.